geode-assembly/src/acceptanceTest/resources/org/apache/geode/client/sni/dual-server-docker-compose.yml

```diff
@@ -43,11 +37,8 @@ services:
       - ./geode-config:/geode/config:ro
       - ./scripts:/geode/scripts
   server-dolores:
-    container_name: 'server-dolores'
```

Comment:
I guess these container names really were redundant since you can refer to the containers by their service name now. ok.

Reply:
And _testcontainers_ errors out if they are left in place...